src/controllers/controller.cpp

```diff
@@ -125,7 +125,8 @@ void Controller::receive(const QByteArray& data, mixxx::Duration timestamp) {
             } else {
                 spacer = QStringLiteral(" ");
             }
-            message += QString::number(data.at(i), 16)
+            // QByteArray is an array of signed 8 bit chars, but QString::number needs an unsigned 8 bit integer
+            message += QString::number(static_cast<quint8>(data.at(i)), 16)
```

Comment:
This is a regression since https://github.com/mixxxdj/mixxx/commit/bd622d3c277591a94f19ecf31fbf3513a3c6c430
and should be fixed independently.

Reply:
Why not merge this PR? Everything is ready!

Reply:
I was not sure about the state here. The review is in progress.

Reply:
Great!